### DISCUSSION TITLE
API POST Size Limit

### DISCUSSION THREAD

**Discussion by jelling** [Comfy-Org/ComfyUI#4003] [Q&A]
Does anyone know the maximum size that can be POSTed to the REST API?

Asking because yesterday I hit an issue where posting Base64 images as inputs caused errors that appeared size related. The same Base64 input nodes worked fine in the interactive mode though. 

I'm working around it for now by sending URLs as inputs. But it would be nice to know if Base64 is an option.

**Comment by VyvaHart** [comment]
Hi @jelling 

There are can be several factors:

- **Server Configuration**: if you have access to the server configuration, check the settings for maximum request size. For example, in Nginx it's - _client_max_body_size_, in Apache - _LimitRequestBody_.
- **Application Settings**: check the config settings of the application framework you are using. For example, in Express.js, you might find settings in _body-parser_
- **Network Components**: load balancers, proxies, or gateways can imposing additional limits

P.S error messages you received might provide specific info about the size limits, but i guess you've checked it first ;)

Good Luck!
